crawl-ref/source/dungeon.cc

```diff
@@ -1338,12 +1338,12 @@ static int _num_mons_wanted()
         return 0;
 
     if (player_in_branch(BRANCH_CRYPT))
```

Comment:
You could simplify this function while you're in here, right? Replace the random2avg for pan with an ordinary NdM dice roll (3d9 looks very close), and remove the doubled call to roll_dice for hell. That way every branch is just a single NdM dice roll, and you can make that explicit by only calling `roll_dice` in one place, with the various branches just having slightly different logic for calculating the number and size of dice to use.

Reply:
Good idea, I'll see if I can do that.